.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.py ident
 *.sh ident
-
+src/dashboard/frontend/app/locale/*.po linguist-generated=true
+src/dashboard/frontend/app/locale/translations.json linguist-generated=true
```

Comment:
Should we also add the following?

    src/dashboard/frontend/app/locale/*.pot linguist-generated=true

Reply:
Done.